packages/terra-application-layout/docs/usage.md

```diff
@@ -0,0 +1,264 @@
+# Terra Application Layout
+
+## Usage Summary
+
+1. The ApplicationLayout requires a `Base` component (from `terra-base`) and any `Router` component (from `react-router-dom`) in its parent hierarchy.
+2. The `nameConfig` prop customizes the ApplicationLayout's branding.
+3. The `utilityConfig` prop generates an utilities menu and presentation controls within the layout. A default set of options can be generated by a helper.
+4. The `routingConfig` prop determines which components get rendered in the ApplicationLayout based on the current router location. Usage of the path `'/'` is restricted for `menu` components.
```

Comment:
terra-utility  menu?

Reply:
`NavigationLayout` menu. The `menu` key is a first-class value used in the `routingConfig`. I can clear this up.